lms/templates/courseware/course_about.html

```diff
@@ -217,6 +217,8 @@ <h1>
 
         <%include file="course_about_sidebar_header.html" />
 
+        ## FIXME add more blocks to simplify the template
```

Comment:
After a ${HTML(parent.body())} in the theme template, some blocks started being visible, but not the block-inside-a-block. I'll debug it more.